linode/resource_linode_instance_ip.go

```diff
@@ -110,6 +110,13 @@ func resourceLinodeInstanceIPCreate(ctx context.Context, d *schema.ResourceData,
 	}
 
 	d.SetId(ip.Address)
+
+	// TODO(jxriddle): check instance status and react to state
```

Comment:
We might want to add some logic here to ensure the instance is `running` to avoid unwanted reboots if an instance is shut down. This will not handle _all_ edge cases, but it should improve the user experience.

Reply:
moved this into the function itself instead of the call